datasets/shield.md

```diff
@@ -1,5 +1,121 @@
+This article introduces the datasets that are useful for analyzing SHIELD studies.
+After reading this article,
+you should understand how to answer questions about
+study enrollment,
+identify telemetry from clients enrolled in an experiment,
+and locate telemetry from add-on studies.
+
+## Table of contents
+
+<!-- toc -->
+
+## Dashboards
+
+[Firefox Test Tube](https://firefox-test-tube.herokuapp.com/)
+shows live enrollment activity and a selection of browser activity metrics
+for active pref-flip (but not add-on) experiments.
+
+## Experiment slugs
+
+Each experiment is associated with a slug,
+which is the label used to identify the experiment to Normandy clients.
+The slug is also used to identify the experiment in telemetry.
+
+You can determine the slug for a particular experiment by consulting
+[Test Tube](https://firefox-test-tube.herokuapp.com/) (only for pref-flip experiments),
+[this summary table](https://metrics.mozilla.com/~sguha/report/normandy_recipes.html),
+or the list of active recipes at
+https://normandy.cdn.mozilla.net/api/v1/recipe/signed/.
+
+## Tables
+
+These tables should be accessible from ATMO, Databricks, Presto, and Athena.
+
+### `experiments` column
+
+[`main_summary`](../datasets/batch_view/main_summary/reference.html),
+[`clients_daily`](../datasets/batch_view/clients_daily/reference.html),
+[`crash_summary`](../datasets/batch_view/crash_summary/reference.html),
+and some other tables
+include a `experiments` column
+which is a mapping from experiment stub to branch.
+
+You can collect rows from enrolled clients using query syntax like:
+
+```sql
+SELECT
+  *,
+  experiments['some-experiment-stub-12345'] AS branch
+FROM clients_daily
+WHERE experiments['some-experiment-stub-12345'] IS NOT NULL
+```
+
+### `experiments`
+
+The `experiments` table is a subset of rows from `main_summary`
+reflecting pings from clients that are currently enrolled in an experiment.
+The `experiments` table has additional string-type
+`experiment_id` and `experiment_branch` columns,
+and is partitioned by `experiment_id`, which makes it efficient to query.
+
+Experiments deployed to large fractions of the release channel
+may have the `isHighVolume` flag set in the Normandy recipe;
+those experiments will not be aggregated into the `experiments` table.
+
+Please note that the `experiments` table cannot be used
+for calculating retention for periods extending beyond
+the end of the experiment.
+Once a client is unenrolled from an experiment,
+subsequent pings will not be captured by the `experiments` table.
+
+### `events`
+
+The [`events` table](../datasets/batch_view/events/reference.html) includes
+Normandy enrollment and unenrollment events.
+
+Normandy events have event category `normandy`.
+The event value will contain the experiment slug.
+
+The event schema is described
+[in the Firefox source tree](https://hg.mozilla.org/mozilla-central/file/c4c125ee2556/toolkit/components/normandy/lib/TelemetryEvents.jsm#l12).
```

Comment:
Re: linking to a specific revision
Pro: the line number doesn't change
Con: the file could change and we'd have no idea that the link was out of date

Not sure what's better here, but I'm kind of inclined to link to latest and have people flag when it's pointing to the wrong thing?

Reply:
Linking the latest sgtm